### PR TITLE
Changed npmrc template name and updated gitignore content

### PR DESCRIPTION
**Pull request by revanth0212**
## Description

Changed .npmrc to npmrc and updated gitignore file contents.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/api-mesh/init.js [hunk 1]
```diff
@@ -86,7 +93,7 @@ class InitCommand extends Command {
 		let shouldCreateGit = gitFlagOptions[flags.git];
 		let packageManagerChoice = flags.packageManager;
 		const packageJsonTemplate = `${getAppRootDir()}/src/templates/package.json`;
-		const dotNpmrcPath = `${getAppRootDir()}/src/templates/.npmrc`;
```
Comment (brasewel) [inline src/commands/api-mesh/init.js:89]: why change from .npmrc to npmrc?

Comment (revanth0212) [reply src/commands/api-mesh/init.js:89]: While publishing `.` files are not published. For instance, here is the latest published package on npm, as you can see none of the `.` files are published

image

Similarly, `template/.npmrc` is also missing which wont let the init command to complete the setup

image

Hence changing it to `npmrc` instead.

**Review by amolina-adobe — APPROVED** [review]

File: src/commands/api-mesh/init.js (hunk 1 above)
Comment (revanth0212) [reply src/commands/api-mesh/init.js:89]: This is from the npm docs, it says `.npmrc` along with a bunch of other files is always ignored https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files

image

Comment (brasewel) [reply src/commands/api-mesh/init.js:89]: ahh. This requires a .npmignore file and you can then specify the files you want in it like so
{
  "files": ["index.js", "/lib"]
}
This whitelists everything inside and you can test it out by running npm pack

Comment (revanth0212) [reply src/commands/api-mesh/init.js:89]: But `.npmrc` is always ignored irrespective of `files` array or `.npmignore`

**Review by brasewel — APPROVED** [review]

**Pull request merged**